### PR TITLE
Create Model, Controller and spec for Cover

### PR DESCRIPTION
**Pull request by ssoriche**
Using `MetaCPAN::Model::User` and `MetaCPAN::Model::Download` as examples, convert `MetaCPAN::Query::Cover` into a model, add the plugin, controller, and tests, and create the OpenAPI spec to drive it.

### CODE REVIEW

File: lib/MetaCPAN/API/Model/Cover.pm [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    my $res = $self->_run_query(
+        index => 'cover',
+        type  => 'cover',
+        body  => {
+            query => $query,
+            size  => 999,
+        }
+    );
+    use DDP;
+    p $res;
```
Comment (oalders) [inline lib/MetaCPAN/API/Model/Cover.pm]: Leftover debugging?

**Comment by oalders** [comment]
This is failing on `t/api/controller/cover.t` in Travis.

File: lib/MetaCPAN/API/Model/Cover.pm (hunk 1 above)
Comment (ssoriche) [reply lib/MetaCPAN/API/Model/Cover.pm]: Oops, trying to do too many things at one time. Cleaning it out in an additional commit that I’ll squash after approval before it gets merged.

**Comment by mickeyn** [comment]
@ssoriche, the test probably fails because there is no data indexed into the `cover` type during `t/0*`.

File: t/api/controller/cover.t [hunk 2]
```diff
@@ -4,6 +4,7 @@ use Test::More;
 use Test::Mojo;
 use Mojo::JSON qw(true false);
 
+$ENV{ES} = 'localhost:9300';
```
Comment (mickeyn) [inline t/api/controller/cover.t]: what's on :9300 ? do we have a new ES instance?

**Comment by ssoriche** [comment]
Yes please. I have a lot of cleanup to do in this branch to remove some "noise".

**Comment by ssoriche** [comment]
Should be good now.

**Pull request merged**